doc/oracle/README.md

```diff
@@ -0,0 +1,122 @@
+Galleon Layers
+=========
+
+* `oracle-datasource`: Provision the `OracleDS` non xa datasource. Depends on `oracle-driver` layer.
+* `oracle-driver`: Provision the `oracle` driver. This layer installs the JBoss Modules module `com.oracle.ojdbc`.
+
+Configuration
+========
+
+The following set of environment variables and corresponding Java system properties can be used to configure the datasource.
+
+Required configuration
+==============
+
+* `ORACLE_PASSWORD`
+
+  * Description: Defines the password for the datasource.
+  * No default value.
+  * Required: True
+  * System Property: `org.jboss.eap.datasources.oracle.password`
+
+* `ORACLE_URL`
+
+  * Description: Defines the connection URL for the datasource.  For example: `jdbc:oracle:thin:@localhost:51521:XE`.
```

Comment:
should we have `_HOST` and `_PORT` env vars (like for the other 2 DBs?)

Reply:
We could but the Oracle JDBC URL has multiple syntax. Not sure we could have the right default value.

Reply:
ok, sounds good.